cardano-testnet/src/Testnet/Components/Configuration.hs

```diff
@@ -81,10 +82,12 @@ numSeededUTxOKeys = 3
 createSPOGenesisAndFiles
   :: (MonadTest m, MonadCatch m, MonadIO m, HasCallStack)
   => CardanoTestnetOptions
+  -> Maybe (ShelleyGenesis StandardCrypto) -- ^ The shelley genesis to use, otherwise a default is picked
```

Comment:
My thoughts are we shouldn't use `Maybe` here. I want to force the caller to be explicit in what genesis they provide.

Reply:
Done :+1: